libs/components/pages/testing/src/link-list/link-list-harness.ts

```diff
@@ -0,0 +1,57 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+import { SkyWaitHarness } from '@skyux/indicators/testing';
+
+import { SkyLinkListHarnessFilters } from './link-list-harness-filters';
+
+/**
+ * Harness for interacting with a link list component in tests.
+ * @internal
```

Comment:
Same here, `@internal`?

Reply:
The `@internal` isn't really needed. Removed.